app/indexers/parent_issue_indexer.rb

```diff
@@ -10,7 +10,7 @@ def to_solr
     return {} unless resource.is_a?(Coin)
     return {} unless parents.first
 
-    parent_attributes.map { |k, v| ["issue_#{k}_tesim", v] }.to_h
+    parent_attributes.map { |k, v| ["issue_#{k}_tesim", Array.wrap(v.to_s)] }.to_h
```

Comment:
These changes concern me given what I'm seeing in https://github.com/pulibrary/figgy/pull/2552/files#diff-1d37e48f9ceff6d8030570cd36286a61R100.  I'll merge, but I'm not certain how the affected tests were passing in the past given that the configurations are identical.

Reply:
Yeah, I commented on this some above. It's weird that this hasn't caused an issue for us before. Solr version diference?